docs/Using Any API/array-response.md

```diff
@@ -0,0 +1,112 @@
+
+```solidity Kovan
+{% include 'samples/APIRequests/FetchFromArray.sol' %}
+```
+
+<div class="remix-callout">
+    <a href="https://remix.ethereum.org/#url=https://docs.chain.link/samples/APIRequests/FetchFromArray.sol" target="_blank" >Open in Remix</a>
+    <a href="/docs/conceptual-overview/#what-is-remix" >What is Remix?</a>
+</div>
+
+To use this contract:
+
+1. Open the [contract in Remix](https://remix.ethereum.org/#url=https://docs.chain.link/samples/APIRequests/FetchFromArray.sol).
+
+1. Compile and deploy the contract using the Injected Web3 environment. The contract includes all the configuration variables for the _Kovan_ testnet. Make sure your wallet is set to use _Kovan_. The _constructor_ sets the following parameters:
+
+   - The Chainlink Token address for _Kovan_ by calling the [`setChainlinkToken`](/docs/chainlink-framework/#setchainlinktoken) function.
+   - The Oracle contract address for _Kovan_ by calling the [`setChainlinkOracle`](/docs/chainlink-framework/#setchainlinkoracle) function.
+   - The `jobId`: A specific job for the oracle node to run. In this case, the _id_ is a _string_ data type, so you must call a job that calls an API and returns a _string_. You can find the job spec for the Chainlink node [here](/docs/direct-request-get-string/).
+
+1. Fund your contract with 0.1 LINK. To learn how to send LINK to contracts, read the [Fund Your Contracts](/docs/fund-your-contract/) page.
+
+1. Call the `id` function to confirm that the `id` state variable is not set.
+
+1. Run the `requestFirstId` function. This builds the `Chainlink.Request` using the correct parameters. The `req.add("path", "0,id")` request parameter tells the oracle node to fetch the id at index 0 of the array returned by the GET request. It uses [JSONPath expression](https://jsonpath.com/) with comma(,) delimited string for nested objects, for example: `'0,id'`.
+
+1. After few seconds, call the `id` function. You should get a non-empty response: _bitcoin_
+
+## Setting the LINK token address, Oracle, and JobId
+
+The [`setChainlinkToken`](/docs/chainlink-framework/#setchainlinktoken) function sets the LINK token address for the [network](/docs/link-token-contracts/) you are deploying to. The [`setChainlinkOracle`](/docs/chainlink-framework/#setchainlinkoracle) function sets a specific Chainlink oracle that a contract makes an API call from. The `jobId` refers to a specific job for that node to run.
+
+Each job is unique and returns different types of data. For example, a job that returns a `bytes32` variable from an API would have a different `jobId` than a job that retrieved the same data, but in the form of a `uint256` variable.
+
+[market.link](https://market.link/) provides a searchable catalogue of Oracles, Jobs and their subsequent return types.
```

Comment:
```suggestion
[market.link](https://market.link/) provides a searchable catalog of oracles, jobs, and their subsequent return types.
```
Americans don't know how to spell "catalogue" 🤣, so let's just use "catalog" consistently across the documentation. Also Oxford comma.

Reply:
hahah sure